brayns/Brayns.cpp

```diff
@@ -138,15 +138,14 @@ struct Brayns::Impl
         if(!geometryParameters.getXYZBFile().empty())
             _loadXYZBFile();
 
-        if(!volumeParameters.getFilename().empty())
+        if(!volumeParameters.getFilename().empty() || !volumeParameters.getFolder().empty())
         {
-            const Vector3ui& volumeDimensions = scene->getVolumeHandler()->getDimensions();
-            const Vector3f& volumeScale = volumeParameters.getScale();
+            const Vector3ui& volumeDimensions = scene->getVolumeHandler()->getDimensions( 0.f );
```

Comment:
I agree. I have to say that I was no so happy myself with that implementation. Simplified :-)